galley/pkg/config/analysis/analyzers/testdata/serviceroleservices.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: details
+  namespace: default
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reviews
+  namespace: default
+---
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: service-role-services-bogus-fqdn
```

Comment:
These names all have the same prefix, which doesn't distinguish them and just makes them a bit harder to read. You can simplify to simply calling them e.g "bogus-fqdn".